### PR TITLE
add Pendulum assets to Nova wallet

### PR DESCRIPTION
**Pull request by b-yap**
Closes https://github.com/pendulum-chain/tasks/issues/365

Added the following assets:

Symbol | Asset Id | ID Scale | Precision | Existential Deposit | Price Id |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
ASTR | [XCM(9)](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/currencyIds.js#L116) | 0x0109  | [18](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer29.js#L308-L310)  | [1000000](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer29.js#L13) | [astar](https://www.coingecko.com/en/coins/astar) | 
| vDOT  | [XCM(10)](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/currencyIds.js#L117)  | 0x010a |  [10](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer36.js#L15)  | [1000000](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer36.js#L16) | [voucher-dot](https://www.coingecko.com/en/coins/voucher-dot) |
| BNC  | [XCM(11)](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/currencyIds.js#L118) | 0x010b |  [12](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer36.js#L49) | [10000000000](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer36.js#L50C27-L50C41) |[ bifrost-native-coin](bifrost-native-coin)
| USDC.axl  | [XCM(12)](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/currencyIds.js#L119) | 0x010c | [6](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer39.js#L25) | [1000](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer39.js#L12) | [axlusdc](https://www.coingecko.com/en/coins/axelar-bridged-usdc) |
| EURC.s (Circle EURC) | [Stellar AlphaNum4](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/currencyIds.js#L70-L78) | 0x0201**45555243**_cf4f5a26e2090bb3adcf02c7a9d73dbfe6659cc690461475b86437fa49c71136_ | [12](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer38.js#L56) | [1000](https://github.com/pendulum-chain/transaction-builder/blob/825e38cba2c77b6a9ba471239dd630a41084beaa/transactions/pendulum_transfer38.js#L12) | [euro-coin](https://www.coingecko.com/en/coins/eurc)

It looks like [USDT has been added already](https://github.com/novasamatech/nova-utils/blob/1067b228dc4dd0ceccb122af1578b0b8057fed35/chains/v20/chains_dev.json#L8055-L8068).

I only updated v20 and v21, because both are the only versions with more assets. I only updated `chain_dev.json`, based on [Gianni's previous PR](https://github.com/novasamatech/nova-utils/pull/2627/files).

### CODE REVIEW

File: chains/v20/chains_dev.json [hunk 1]
```diff
@@ -8214,6 +8214,76 @@
...
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 18,
+                "symbol": "EURC.s",
+                "precision": 12,
+                "type": "orml",
+                "priceId": "euro-coin",
+                "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/tokens/white/EUR.svg",
```
Comment (ebma) [inline chains/v20/chains_dev.json]: The icon here is for Mykobo's EURC. We should use the icon for the EURC issued by circle.

Comment (ebma) [reply chains/v20/chains_dev.json]: @prayagd I can see that the [linked icon folder](https://drive.google.com/drive/u/0/folders/1sZcQzoDxT_EUd2wtEJN8ECNzPakfAMs6) only has the EURC.svg for Mykobo. Can we also have the Circle EURC icon as an SVG please?

**Review by ebma — COMMENTED** [review]
Looks good to me, except for that one icon 👍

File: chains/v20/chains_dev.json (hunk 1 above)
Comment (ebma) [reply chains/v20/chains_dev.json]: Pinging again

Comment (prayagd) [reply chains/v20/chains_dev.json]: Sorry somehow missed this message, will share it here

Comment (prayagd) [reply chains/v20/chains_dev.json]: https://drive.google.com/file/d/17XHmn9qpkNcZbqPyRhw2D2C8B1dP9xJ1/view?usp=drive_link

Comment (prayagd) [reply chains/v20/chains_dev.json]: i already had a EURC circle.svg but nevermind

**Comment by ebma** [comment]
I added the other icon and will now create a pull request to the upstream repository.

**Pull request closed without merging**